Samples.Swift/Samples.Swift/ViewControllers/CheckoutViewController/CheckoutViewController.swift

```diff
@@ -0,0 +1,463 @@
+
+          guard
+            let data = data,
+            let tokenResponse = try? PayPal.jsonDecoder.decode(AccessTokenResponse.self, from: data)
+          else {
+            print("Fetch access token failed with no data")
+            return
+          }
+
+          PayPal.shared.setAccessToken(tokenResponse.accessToken)
+
+          PayPal.shared.request(on: .createOrder(order)) {
+            data, error in
+            guard
+              let data = data,
+              let createOrderResponse = try? PayPal.jsonDecoder.decode(CreateOrderResponse.self, from: data)
+            else {
+              print("Create order failed with no data")
+              return
+            }
+
+            action.set(orderId: createOrderResponse.id)
+          }
+        }
+      })
+    case 2:
+      // TODO: implement payment buttons logic
+    break
+
+    default:
+      break
+    }
+  }
+
+  func getOrder() -> OrderRequest {
+    var itemsList = [PurchaseUnit.Item]()
+
+    for item in items {
+      let item = PurchaseUnit.Item(
+        name: item.name,
+        unitAmount: UnitAmount(currencyCode: .usd, value: item.unitAmount.value),
+        quantity: item.quantity,
+        tax: PurchaseUnit.Tax(currencyCode: .usd, value: String(item.tax?.value ?? ""))
+      )
+      itemsList.append(item)
+    }
+
+    let purchaseUnit: PurchaseUnit = PurchaseUnit(
+      amount: PurchaseUnit.Amount(
+        currencyCode: .usd,
+        value: String(self.getTotal()),
+        breakdown: PurchaseUnit.Breakdown(
+          itemTotal: UnitAmount(
+            currencyCode: .usd,
+            value: String(self.getItemTotal())
+          ),
+          taxTotal: UnitAmount(
+            currencyCode: .usd,
+            value: String(self.getTaxTotal())
+          )
+        )
+      ),
+      items: itemsList
+    )
+
+
+    let order: OrderRequest = OrderRequest(
+      intent: .authorize,
+      purchaseUnits: [purchaseUnit]
+    )
+
+    return order
+  }
+
+  func setupTableView() {
+    tableView.register(AddItemCell.self, forCellReuseIdentifier: "AddItemCell")
+    tableView.register(ItemCell.self, forCellReuseIdentifier: "ItemCell")
+    tableView.register(TotalCell.self, forCellReuseIdentifier: "TotalCell")
+
+    tableView.delegate = self
+    tableView.dataSource = self
+  }
+
+  func setupSampleItem() {
+    let item: PurchaseUnit.Item = PurchaseUnit.Item(
+      name: "Sample item",
+      unitAmount: UnitAmount(
+        currencyCode: .usd, value: "5.99"
+      ),
+      quantity: "1",
+      tax: PurchaseUnit.Tax(
+        currencyCode: .usd,
+        value: "1.23"
+      )
+    )
+
+    items.append(item)
+  }
+
+  func setupUI() {
+    view.backgroundColor = .white
+    let checkoutFlowOptions = ["Order", "ECToken", "Payment Button"]
+    checkoutFlowOption = UISegmentedControl(items: checkoutFlowOptions)
+    checkoutButton.setTitle("Checkout with Cart", for: .normal)
```

Comment:
nitpicking haha, but maybe we can change this to "Checkout with Order" for consistency with what we have in the segmented control.